app/locale/en_US/Mage_Adminhtml.csv

```diff
@@ -569,11 +569,10 @@
 "Magento Connect","Magento Connect"
 "Magento Connect Manager","Magento Connect Manager"
 "Magento Logo","Magento Logo"
-"Magento is a trademark of Magento Inc. Copyright &copy; %s Magento Inc.","Magento is a trademark of Magento Inc. Copyright &copy; %s Magento Inc."
+"Magento is a trademark of Magento Inc. Copyri ght &copy; %s Magento Inc.","Magento is a trademark of Magento Inc. Copyright &copy; %s Magento Inc."
 "Magento root directory","Magento root directory"
-"OpenMage ver. %s","OpenMage ver. %s"
+"OpenMage LTS ver. %s","OpenMage LTS ver. %s"
```

Comment:
Can we remove the `ver.` string?

Agree with @Flyingmana to show current M1 version .....


Prefered outpu `OpenMage LTS 19.4.3 (based on Magento 1.9.4.5)`

Reply:
Reasons to remove the Magento version from footer:

1. After M1 is EOL referencing Magento is just referencing an unsupported product, this doesn't look good in the eyes of some people as it relates to PCI compliance.
2. If removing the trademark notice then all references to "Magento" should also be removed (though I'm not a legal expert).
3. Users of OpenMage are highly likely to know enough about it to know the version it is based on so don't need a reminder every day :)

Reply:
@sreichel What do you think?

Reply:
@colinmollenhour i dont know. At the one hand I understand Daniel, at the other i agree with your 3rd. (and finaly to 1st and 2nd). Just remove the `ver.` string and i'm happy with :)

Reply:
M1 is EOL in some days ... lets drop that "based on" .... and remove "ver."! ;)

Reply:
```suggestion
"OpenMage LTS %s","OpenMage LTS %s"
```